modules/ingestor/src/graph/sbom/spdx.rs

```diff
@@ -1,3 +1,4 @@
+use crate::graph::sbom::NodeInfoParam;
```

Comment:
Please nest the imports.

Reply:
ok

Reply:
@bxf12315 it looks to me like the same should be applied in https://github.com/trustification/trustify/pull/1353/files#diff-765ccf6c9e29cbd1174c4c02a92a18501c9650a4b520f58422694b84956e1113R1